src/scss/_hds.scss

```diff
@@ -2,11 +2,6 @@
 // @file
 // This file includes Helsinki Design System core packages.
 //
```

Comment:
Don't get scared now. The normalize.css and necessary design tokens are actually already loaded in hds-core/.../base.css!